pkg/trace/traceutil/azure.go

```diff
@@ -34,7 +35,8 @@ const (
 	containerFramework = "Container"
 	unknown            = "unknown"
 
-	appService = "app"
+	appService  = "app"
+	functionApp = "functionapp"
```

Comment:
Why make this a constant? It's only used on line 93, and seems reasonable to just have there.